docs/source/advanced/test-builders.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: 'Using models in tests'
+---
+
+Apollo Android generates model classes based on your schema and operations. You can then use these classes in your
```

Comment:
Nit: I'd remove `schema`. The schema is used for validation but doesn't really influence the model shapes. 

```suggestion
Apollo Android generates model classes based on your operations. You can then use these classes in your
```